bin/run_hvac.py

```diff
@@ -26,7 +26,7 @@
 from lsst.ts import hvac
 
 logging.basicConfig(
-    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.INFO,
+    format="%(asctime)s:%(levelname)s:%(item)s:%(message)s", level=logging.INFO,
```

Comment:
Indeed a bug. Occasionally PyCharm refactors too much when I rename a variable and it is quite difficult to revert the unwanted changes. Thanks for catching this one.

And I basically added this for my convenience during software development so I can see what the CSC does. I'll investigate a way to add logging to stdout to the existing logger instead of replacing it.